docs/variants/pc_engines/releases_seabios.md

```diff
@@ -0,0 +1,19 @@
+# Release Notes
+
+Following Release Notes describe status of Dasharo SeaBIOS variant of
+open-source firmware development for PC Engines apu2 series platform.
```

Comment:
Shouldn't we say precisely about apu2/3/4/6?

Reply:
Maybe

Reply:
https://github.com/Dasharo/docs/pull/794/commits/d88e1db523898f4842a0cd51f25a9bff02b99563